doc/design.md

```diff
@@ -165,6 +165,28 @@ Covers:
 
 Needs: impl
 
+### RR Creates Validation Report
+`dsn~rr-creates-validation-report~1`
+
+RR creates a validation report containing a summary of all validations and their results.
+
+Covers:
+
+* `req~validation-report~1`
+
+Needs: impl, utest
+
+### RR Writes Validation Report to a File
+`dsn~rr-writes-validation-report-to-file~1`
+
+RR writes validation report to a `home/.release-robot/last_report.txt` file.
```

Comment:
We already have a folder .release-robot because we store credentials there. I thought it's better to have one place, isn't it? And I don't really want to delete it, as it's just one file, it won't hurt to store it